src/Startup/Backend/Monitor.vala

```diff
@@ -49,14 +49,19 @@ public class Startup.Backend.Monitor : Object, Port.Monitor {
     void on_change_occurred (File file, File? dest, FileMonitorEvent event) {
         var path = file.get_path ();
 
-        if (Utils.is_desktop_file (path) == false)
+        if (Utils.is_desktop_file (path) == false) {
             return;
+        }
 
-        if (event == FileMonitorEvent.CREATED)
-            file_created (path);
-        else if (event == FileMonitorEvent.DELETED)
-            file_deleted (path);
-        else if (event == FileMonitorEvent.CHANGED)
-            file_edited (path);
+        switch (event) {
+            case FileMonitorEvent.CREATED:
+                file_created (path);
+                break;
+            case FileMonitorEvent.DELETED:
+                file_deleted (path);
+                break;
+            case FileMonitorEvent.CHANGED:
+                file_edited (path);
```

Comment:
A `break` is missing here?

Reply:
I was about to ask that on slack. Since it is the last case is break really necessary here? I have seen break being used in default case too in the code base

Reply:
I added it. valac complains about it :)

Reply:
We can't omit `break` in Vala like C#, otherwise [the CI fails](https://travis-ci.com/elementary/switchboard-plug-applications/builds/127790106#L585-L586).